server/src/test/java/org/opensearch/ingest/IngestServiceTests.java

```diff
@@ -2058,6 +2059,18 @@ public void testPrepareBatches_different_index_pipeline() {
         assertEquals(4, batches.size());
     }
 
+    public void testUpdateMaxIngestProcessorCountSetting() {
```

Comment:
Can you please add the UT for the `validateProcessorCountForIngestPipeline` as well? 

You can take some reference from the other validatePipeline tests.

https://github.com/opensearch-project/OpenSearch/blob/8629279533ff1a2f2c7bfcd772382d8b50d5cfba/server/src/test/java/org/opensearch/ingest/IngestServiceTests.java#L368-L369

Reply:
Added `validateProcessorCountForIngestPipeline`